client/packages/common/src/ui/layout/tables/types.ts

```diff
@@ -20,6 +20,8 @@ export interface TableProps<T extends RecordWithId> {
   data?: T[];
   dense?: boolean;
   ExpandContent?: FC<{ rowData: T }>;
+  // specify a key to enable the option to hide columns
```

Comment:
would prefer `enableColumnSelection` or similar - but also need a unique identifier for the table. Munged the two together, which isn't ideal

Reply:
could have tableId or tableName or tableKey, and enableColumnSelection

Reply:
sigh. yes - was avoiding updating all instances of `DataTable`, but it is better.